Quick-Start [DA]/Sample-Ext.md

```diff
@@ -1 +1,259 @@
+-- 1) Insert a row with a value from the composite type
+INSERT INTO my_table (person) VALUES (('John Doe', 30));
+
+-- Output:
+ id |     person
+----+-----------------
+  1 | ("John Doe",30)
+(1 row)
+
+-- 2) Insert a row with a value from the enumerated type
+INSERT INTO status_table (status) VALUES ('active');
+
+-- Output:
+ id | status
+----+--------
+  1 | active
+(1 row)
+
+```
+
+In the example above, we demonstrate how to use the `person_type` composite type to declare a variable and perform operations using its attributes. We also showcase the usage of the `status_type` enumerated type in a table column, ensuring that only the predefined values ('active', 'inactive', 'pending') can be inserted.
+
+### 2. `CREATE DOMAIN` statement:
+- A domain type is a custom data type that is based on an existing base type but with additional constraints or rules applied. It provides a way to create a specialized version of an existing data type.
+- Domains are useful for adding constraints or business rules to a base type without duplicating those constraints across multiple columns.
+
+A domain type using `CREATE DOMAIN`:
+```sql
+-- Create a domain type for representing positive integers
+CREATE DOMAIN positive_integer AS integer
+   CHECK (VALUE > 0);
+```
+
+In the example above, we create a domain type called `positive_integer` based on the existing base type `integer`. The `CHECK` constraint ensures that any value assigned to this domain type must be greater than 0, effectively enforcing the concept of positive integers.
+
+By using the domain type `positive_integer`, you can apply this constraint to multiple columns or variables without the need to duplicate the constraint definition each time. It promotes consistency and simplifies the management of constraints by encapsulating them within the domain type.
+
+For more detailed information on `CREATE DOMAIN`, you can refer to the [PostgreSQL documentation on `CREATE DOMAIN`](https://www.postgresql.org/docs/current/sql-createdomain.html).
+
+Usage example:
+```sql
+-- Create a table using the domain type
+CREATE TABLE quant_table (
+   id serial PRIMARY KEY,
+   quantity positive_integer
+);
+
+-- Insert a row, ensuring the quantity is a positive integer
+INSERT INTO quant_table (quantity) VALUES (10);
+
+-- Output:
+ id | quantity
+----+----------
+  1 |       10
+(1 row)
+
+-- This insert will fail due to the CHECK constraint
+INSERT INTO my_table (quantity) VALUES (-5);
+
+-- Output:
+ERROR:  value for domain positive_integer violates check constraint "positive_integer_check"
+```
+
+## Custom Postgres Operators:
+[3rd Party Site](https://www.linuxtopia.org/online_books/database_guides/Practical_PostgreSQL_database/PostgreSQL_x15284_002.htm)
+
+In PostgreSQL, custom operators can be created to extend the functionality of the database by defining new operations or modifying the behaviour of existing operations.
+
+```sql
+-- Create necessary objects for version 1.0.2
```

Comment:
Version 1.0.2 of what?

Reply:
this is a typo. I was trying something else. I'll fix this ASAP.